<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClassify</class>
 <widget class="QDialog" name="DialogClassify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地物分类</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>591</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_outputDir"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton_outputDir">
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>输出文件夹</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_inputImg">
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>输入影像</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_inputImg"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonOK">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
